nixos/modules/system/boot/loader/raspberrypi/raspberrypi-builder.sh

```diff
@@ -1,4 +1,4 @@
-#! @bash@/bin/sh
+#! @bash@/bin/bash -e
```

Comment:
In #90119 it was decided to remove `-e` which is redundant because the `set -e` coming right after. (See the resolved discussion I can't link to because it's resolved in that PR.)

In some situations the script ends up being called in a way where the shebang is disregarded.

Reply:
Ah I saw this, but it must have gotten clobbered when I copied over the extlinux script. Should I go in and fix it in the extlinux script too while I'm at it?

Reply:
No need to, it have to be checked whether it's running with or without `-e` enabled properly beforehand, and it expands the scope of this PR needlessly.